tests/profiles/adobefonts_test.py

```diff
@@ -95,8 +124,7 @@ def test_check_nameid_1_win_english():
         adobefonts_profile, "com.adobe.fonts/check/nameid_1_win_english"
     )
 
-    font = TEST_FILE("source-sans-pro/OTF/SourceSansPro-Regular.otf")
-    ttFont = TTFont(font)
```

Comment:
yeah, I also like doing it all in a single (whenever possible) just like you did here ;-)

Reply:
👍